### PR TITLE
Fix duplicate article when added via the bookmarklet

### LINKED ISSUE #1886: Duplicate entry via bookmarklet
### Issue details

When an article already present is added via the "+" button, a message indicate that it was already added and the date.
When adding the same article via the bookmarklet, a new entry is created 

### Environment

* wallabag version (or git revision) that exhibits the issue: 2.0.0
* Wallabag installation via git clone
* php version: 5.6.19-0+deb8u1
* OS: debian 8.4
* type of hosting (shared or dedicated): dedicated VM
* which storage system you choose at install (SQLite, MySQL/MariaDB or PostgreSQL): PostgreSQL

### Steps to reproduce/test case

Open the original article of an entry present in wallabag. Use the bookmarklet to save it. A new entry is present and is a duplicate (the previous entry is still present).

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| Tests pass?   | yes
| Fixed tickets | #1886
| License       | MIT

Fix #1886

@j0k3r can you tell me if we need to refactor with `addEntryFormAction`? And do we need tests here?

### CODE REVIEW

**Comment by j0k3r** [comment]
Yeah both :)

File: src/Wallabag/CoreBundle/Controller/EntryController.php [hunk 1]
```diff
@@ -420,4 +422,16 @@ private function checkUserAction(Entry $entry)
...
     }
+
+    /**
+     * Check for existing entry, if it exists, redirect to it with a message.
+     *
+     * @param $entry
+     *
+     * @return array|bool
+     */
+    public function checkIfEntryAlreadyExists($entry)
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Controller/EntryController.php]: I'll put that private

**Comment by j0k3r** [comment]
Ok test aren't needed since the new method is shared.

File: src/Wallabag/CoreBundle/Controller/EntryController.php (hunk 1 above)
Comment (nicosomb) [inline src/Wallabag/CoreBundle/Controller/EntryController.php]: Oh yes it was what I wanted to do (but phpstorm refactored with a public method ... and I didn't checked!)

**Pull request merged**